code/modules/mob/living/emote.dm

```diff
@@ -287,14 +294,18 @@
 	return ..() && user.can_speak(allow_mimes = TRUE)
 
 /datum/emote/living/laugh/get_sound(mob/living/user)
-	if(!ishuman(user))
+	if(!ishuman(user) || !islizard(user) || !ismoth(user))
```

Comment:
Lizards and Moths are Humans as far as `ishuman` is concerned.

Reply:
why do we need a check for islizard and ismoth if ishuman checks for them already then?

Reply:
ah, MY code was the one doing that, I'll remove it thanks.